src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -428,9 +432,7 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       sectional: {
         inputType: 'options',
         skip: true,
-        defaultValue: false,
-        hidden: false,
-        required: true,
+        required: false,
```

Comment:
This is the new way for optional args: just `required: false`. Sweep's sectional was our first test case here in this PR.